Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -345,16 +345,16 @@ void CDMInstanceOpenCDM::loadSession(LicenseType, const String& sessionId, const
     std::string responseMessage;
     SessionLoadFailure sessionFailure = SessionLoadFailure::None;
 
-    if (!session->load(responseMessage)) {
-        if (!responseMessage.compare(0, 8, "message:")) {
-            GST_TRACE("message length %u", responseMessage.size());
-            callback(std::nullopt, std::nullopt, std::nullopt, SuccessValue::Succeeded, sessionFailure);
-        } else {
+    if (session->load(responseMessage)) {
+        if (responseMessage.compare(0, 8, "message:")) {
```

Comment:
Ditto about the isMessage checks

Reply:
I have the impression that you didn't check the whole diff, did you?

Reply:
I can assure you I spent time going through all the commits, you did force push the branch in the middle of my review.

Reply:
I did, but I was quite quick! :angel: